make_figures.py

```diff
@@ -195,7 +195,8 @@ def make_timeplot(df_measure, df_prediction):
             showarrow=False,
             font_size=LABEL_FONT_SIZE - 6,
             font_color="DarkSlateGray",
-            text="Drag handles below to change time window")
+            text="Drag handles below to change time window",
+            align="center")
```

Comment:
Definitely seems like an improvement. Shouldn't we right-align and put x=1? That way it's more likely to be on top of the handles.

Reply:
I see what you mean.